assets/src/blocks/TableOfContents/TableOfContentsFrontend.js

```diff
@@ -0,0 +1,19 @@
+import {getTableOfContentsStyle} from './getTableOfContentsStyle';
+import {TableOfContentsItems} from './TableOfContentsItems';
+import {makeHierarchical} from './makeHierarchical';
+import {getHeadingsFromDom} from './getHeadingsFromDom';
+
+export const TableOfContentsFrontend = ({title, className, levels, submenu_style}) => {
+  const headings = getHeadingsFromDom(levels);
+  const menuItems = makeHierarchical(headings);
+  const style = getTableOfContentsStyle(className, submenu_style);
+
+  return (
+    <section className={`block table-of-contents-block table-of-contents-${style} ${className ?? ''}`}>
+      {!!title && (
```

Comment:
This can be simplified to a one-liner: `{!!title && <h2>{title}</h2>}`

Reply:
Done!